src/__docs__/index.html

```diff
@@ -67,6 +66,37 @@
       --calendar-main-color: #263340;
       --viewport-border-thickness: 1px;
     }
+
+    pre {
+      display: block;
+      padding: 9.5px;
+      margin: 0 0 10px;
+      font-size: 13px;
+      line-height: 1.42857143;
+      color: #333;
+      word-break: break-all;
+      word-wrap: break-word;
+      background-color: #f5f5f5;
+      border: 1px solid #ccc;
+      border-radius: 4px;
+    }
+
+    button, input, select, textarea {
+      font-family: inherit;
+      font-size: inherit;
+      line-height: inherit;
+    }
+
+    .row {
+      margin-right: -15px;
+      margin-left: -15px;
+    }
+
+    * {
+      -webkit-box-sizing: border-box;
+      -moz-box-sizing: border-box;
+      box-sizing: border-box;
+    }
```

Comment:
Do these styles need to be bundled with our output? Or are they just resets to protect us from docz values with our components?

Reply:
Good question, if we need to guarantee these components to be totally separated from bootstrap dependency, we have to have these styles in the bundle.